packages/uploadthing/src/client.ts

```diff
@@ -1,14 +1,17 @@
-/* eslint-disable no-console -- Don't ship our logger to client, reduce size*/
-
 import type { ParseError } from "@effect/schema/ParseResult";
-import * as S from "@effect/schema/Schema";
+import * as Array from "effect/Array";
```

Comment:
did this need to be aliased to something other than `Array` to work in react native? 

https://discord.com/channels/795981131316985866/1233104222954852454/1233174648573788200

not sure if it's the case when we do the `import * as Array` syntax though, I assume it will?

Reply:
Ah good catch, force of habit from my own apps-- which are not react-native :)
`Arr` is fine, yes? `A` is also common